core/src/main/java/com/opensymphony/xwork2/ActionContext.java

```diff
@@ -157,6 +129,15 @@ public static ActionContext of(Map<String, Object> context) {
         return new ActionContext(context);
     }
 
+    /**
+     * Creates a new ActionContext based on empty Map
+     *
+     * @return new ActionContext
+     */
+    public static ActionContext of() {
```

Comment:
`create()` might be a more descriptive name for this method

Reply:
There is already `of(Map<String, Object> context)` so I stick to the same convention